docs/diginstroom/sip/1.0/sip_structure/5_structure_package.md

```diff
@@ -1818,6 +1821,31 @@ TODO: figure out the IDs
 | Cardinality | 1..1 |
 | Obligation | MUST |
 
+| Element | `premis:premis/premis:event/premis:eventOutcomeInformation` |
+|-----------------------|-----------|
+| Name | Event outcome information  |
+| Description | Information about the outcome of an event. |
+| Cardinality | 0..* |
+| Obligation | MAY |
+
+| Element | `premis:premis/premis:event/premis:eventOutcomeInformation/premis:eventOutcome` |
+|-----------------------|-----------|
+| Name | Event outcome  |
+| Description | This element categorizes the outcome of the event in terms of success or failure. |
+| Datatype | [String]({{ site.baseurl }}{% link docs/diginstroom/sip/1.0/2_terminology.md %}#string); fixed vocabulary (e.g. [`PREMIS event outcome`](https://id.loc.gov/vocabulary/preservation/eventOutcome.html)) |
+| Vocabulary | `fail`<br>`success`<br>`warning` |
+| Cardinality | 1..1 |
```

Comment:
Wou jij hier geen MAY? Persoonlijk vind ik het wel beter zo om geen ambiguïteit over de status te introduceren, ook al zijn we hier strenger dan PREMIS